stories/DayPicker.js

```diff
@@ -52,12 +52,11 @@ storiesOf('DayPicker', module)
   .addWithInfo('with custom day size', () => (
     <DayPicker daySize={50} />
   ))
-  .addWithInfo('more than one month', () => (
```

Comment:
We should have all of explicit 1, > 1, and "default". That way we don't have to update this in the future if the default changes.